From c239d623d50a5e603427923b77d1e3ddcb4b5ab8 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Thu, 2 Oct 2025 07:45:46 +0200
Subject: [PATCH] target: add fw3 default packages variant

Add a new DEFAULT_PACKAGES.fw3 variant that uses firewall3 instead of
firewall4. This is needed for the mediatek-sdk kernel which is based on
kernel 5.4.271 and requires firewall3 compatibility.

Signed-off-by: John Crispin <john@phrozen.org>
---
 include/target.mk | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/include/target.mk b/include/target.mk
index 637ef2077e..d830105106 100644
--- a/include/target.mk
+++ b/include/target.mk
@@ -71,6 +71,18 @@ DEFAULT_PACKAGES.qsdk:=\
         ppp-mod-pppoe \
 	 -procd-ujail
 
+DEFAULT_PACKAGES.fw3:=\
+        dnsmasq \
+        firewall \
+        ip6tables \
+        iptables \
+        kmod-ipt-offload \
+        odhcp6c \
+        odhcpd-ipv6only \
+        ppp \
+        ppp-mod-pppoe
+
+
 ifneq ($(DUMP),)
   all: dumpinfo
 endif
-- 
2.34.1

